client/src/components/auth/ProtectecRoute.js

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from "react";
```

Comment:
You've misspelled this file, `ProtectedRoute.js` not `ProtectecRoute.js` ;)

Reply:
Also use this with your `Tracker.js`. This should be the mechanism you reuse on any route that you must be logged in to view

Reply:
How do I use this with my Tracker.js ?

Reply:
Take a look at this https://tylermcginnis.com/react-router-protected-routes-authentication/

Reply:
Or the docs are more concise: https://reacttraining.com/react-router/web/example/auth-workflow